apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/PathExpectationSamples.kt

```diff
@@ -29,12 +28,362 @@ class PathExpectationSamples {
 
     @Test
     fun toBeAnEmptyDirectory() {
-        val dir = tempDir.newDirectory("dir")
+        val dir = tempDir.newDirectory(TEST_DIR)
         expect(dir).toBeAnEmptyDirectory()
 
-        dir.newFile("test.txt")
+        dir.newFile("$TEST_FILE.txt")
         fails {
             expect(dir).toBeAnEmptyDirectory()
         }
     }
+
+    @Test
+    fun toStartWith() {
+        val dir = tempDir.newDirectory(TEST_DIR)
+
+        expect(dir.last()).toStartWith(Paths.get(TEST_DIR))
```

Comment:
This does not show the toStartWith functionality nicely -- in the end your check does the following: `expect(Paths.get("test_dir")).toStartWith(Paths.get("test_dir"))`.
I suggest you rewrite it to something like the following. 
```suggestion
        expect(dir).toStartWith(dir.parent)
```
Same same for notToStartWith/ (not)toEndWith